@shared/api/types.ts

```diff
@@ -31,6 +31,7 @@ export interface Response {
   isTestnet: boolean;
   isMemoValidationEnabled: boolean;
   isSafetyValidationEnabled: boolean;
```

Comment:
I'm thinking we should change `isSafetyValidationEnabled` to something like `isValidatingSafeAccountsEnabled`. I don't _think_ there will be an issue changing this localStorage key. We could default to true for the new key

I can create a ticket for if agreed

Reply:
Yeah, this is a good call. This was an oversight on my part. I should have made this key name more specific to begin with 👍

Reply:
https://stellarorg.atlassian.net/browse/WAL-477?atlOrigin=eyJpIjoiZTE1OGVhMTVhYzFmNGIxOTk4NDczOWJkNTM4NTU0MGMiLCJwIjoiaiJ9